roomserver/storage/sqlite3/deltas/20221027084407_published_appservice.go

```diff
@@ -24,8 +24,8 @@ func UpPulishedAppservice(ctx context.Context, tx *sql.Tx) error {
 	_, err := tx.ExecContext(ctx, `	ALTER TABLE roomserver_published RENAME TO roomserver_published_tmp;
 CREATE TABLE IF NOT EXISTS roomserver_published (
     room_id TEXT NOT NULL,
-    appservice_id TEXT NOT NULL,
-    network_id TEXT NOT NULL,
+    appservice_id TEXT,
+    network_id TEXT,
```

Comment:
i don't believe these 2 fields can be null. appservice should always have an id. Don't know what networkid is. If it is the url, then that is also a required field. I've been running sqlite until recently when Brian created the postgresql docker - after which I updated my dendrite.yaml to use postgresql.  In fact, I had switched back to sqlite yesterday to test my fix for an issue. Didn't run into this issue. Something else might be going on. Let's hop on a call.

Reply:
👍